quickjs.c

```diff
@@ -1932,6 +1948,11 @@ void JS_SetGCThreshold(JSRuntime *rt, size_t gc_threshold)
     rt->malloc_gc_threshold = gc_threshold;
 }
 
+void JS_SetHooksHandler(JSRuntime *rt, JS_BOOL (*fn)(JSRuntimeHooks type, void* opaque))
```

Comment:
Why take an opaque if you are not doing anything with it? An opaque would make sense if you support multiple callbacks, so each could get their own.